commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2SubCommand.java

```diff
@@ -41,13 +45,34 @@
       arity = "1")
   private boolean slashingProtectionEnabled = false;
 
+  @Option(
+      names = {"--slashing-db-url"},
+      description = "The jdbc url to use to connect to the slashing database",
+      paramLabel = "<storage label>",
+      arity = "1")
+  private String slashingDbUrl;
+
+  @Option(
+      names = {"--slashing-db-username"},
+      description = "The username to use when connecting to the slashing database")
+  private String slashingDbUser;
+
+  @Option(
+      names = {"--slashing-db-password"},
+      description = "The password to use when connecting to the slashing database")
+  private String slashingDbPassword;
```

Comment:
is it ok to specify password on cli? or should we instead specify a password file.

Reply:
I dont think (I might be wrong though) we are using config file/env variable in web3signer (the change was in ethsigner).

Reply:
There's a YamlConfigFileDefaultProvider and EnvironmentVariableDefaultProvider so it should be working for web3signer.